spi/topology/src/main/java/quarks/topology/spi/graph/TWindowTimeImpl.java

```diff
@@ -94,5 +94,33 @@ Licensed to the Apache Software Foundation (ASF) under one
         
         Aggregate<T,U,K> op = new Aggregate<T,U,K>(window, batcher);
         return feeder().pipe(op); 
+    }
+
+    /**
+     * @return the time
+     */
+    public long getTime() {
+        return time;
+    }
+
+    /**
+     * @param time the time to set
+     */
```

Comment:
windowing impl newbie wondering about the addition of the setters...  the ctor requires the values be supplied.  Is it really necessary to make them changeable post construction?  They can't be made final?  Changing the values after calling aggregate() or batch() seems to have no affect on the behavior of the windows associated with them.  Maybe just WIP, maybe just need more doc stating that, maybe some class doc clarifying usage of the class - e.g., is a TWindowTimeImpl instance supposed to represent a single window instance or a factory for one or more actual windows (i.e., it's legal to call both aggregate() and batch(), or each one more than once)?